src/main/scala/uk/gov/hmrc/common/message/model/package.scala

```diff
@@ -28,15 +28,15 @@ package object model {
   val defaultJodaDateFormat = "yyyy-MM-dd"
 
   def formatDate(date: LocalDate): String =
-    "%04d-%02d-%02d".format(date.getYear, date.getMonthOfYear, date.getDayOfMonth)
+    "%04d-%02d-%02d".format(date.getYear, date.getMonthValue, date.getDayOfMonth)
 
-  def jodaDateReads: Reads[LocalDate] =
+  def localDateReads: Reads[LocalDate] =
     Reads[LocalDate](
       js =>
         js.validate[String]
           .map[LocalDate](dtString =>
             Try {
-              LocalDate.parse(dtString, DateTimeFormat.forPattern(defaultJodaDateFormat))
+              LocalDate.parse(dtString, DateTimeFormatter.ofPattern(defaultJodaDateFormat))
```

Comment:
Very  minor point, but this still says Joda in the name. In fact, its just a dateFormat.

Reply:
good one, it is just existing name, will update it anyway. thanks @ardwalker